src/pages/keyblocks/[id].vue

```diff
@@ -49,3 +50,12 @@ await useAsyncData(async() => {
   return true
 })
 </script>
+
+<style scoped>
+.keyblocks_keyblock-details-panel {
```

Comment:
Maybe `keyblock-details__keyblock-details-panel` is more precise?

Reply:
Absolutely, ser. Fixed